.github/workflows/nodejs.yml

```diff
@@ -12,20 +12,13 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [12.x]
-
     steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: 16
       - run: sudo apt-get install xvfb
       - run: npm ci
       - run: npm run build --if-present
-      - run: xvfb-run --auto-servernum npm run test
```

Comment:
You can get away without this?

Reply:
We don't have any tests and `npm run test` is actually not defined in the `package.json`.